docs/hyperloop/userdocumentation.md

```diff
@@ -259,7 +277,64 @@ When creating or enabling wagons, you can use a pull request instead of a packag
     <img src="../images/graphZoom2.png" width="70%">
   </div>
 
+* In _Submitted jobs_, you can see the summary of the master jobs, along with links to the **IO Statistics** and **Stack trace**.
+
+<div align="center">
+   <img src="../images/submittedJobs1.png" width="70%">
+</div>
+
+* Click the **IO Statistics** button to be redirected to the site activity information.
+
+<div align="center">
+   <img src="../images/submittedJobs2.png" width="70%">
+</div>
+
+* Click the **Stack trace** button to be redirected to the stack trace information in MonALISA.
```

Comment:
Add: "Here you can see a summary of failures of your jobs. This information is collected when the masterjobs have finished from all ERROR_V jobs. Some information is already available while the train is running but make sure to check again when the train is in a final state. Common errors are grouped and counted. This allows you to investigate failures and debug them using the provided stack trace."